etc/bento.env

```diff
@@ -236,14 +236,14 @@ BENTOV2_DRS_EXTERNAL_PORT=7007
 BENTO_DRS_CONTAINER_DATA_VOLUME_DIR=/drs/bento_drs/data
 BENTO_DRS_CONTAINER_TMP_VOLUME_DIR=/drs/tmp
 
-BENTOV2_DRS_PROD_VOL_DIR=${BENTOV2_ROOT_DATA_DIR}/drs/data
+BENTOV2_DRS_PROD_VOL_DIR=${BENTO_SLOW_DATA_DIR}/drs/data
 BENTOV2_DRS_PROD_WES_VOL_DIR=${BENTOV2_WES_VOL_TMP_DIR}
 # ( assumes a relative wes path like ./lib/wes/... - adjust accordingly )
 
-BENTOV2_DRS_DEV_VOL_DIR=${BENTOV2_ROOT_DATA_DIR}/drs/data
+BENTOV2_DRS_DEV_VOL_DIR=${BENTO_SLOW_DATA_DIR}/drs/data
 BENTOV2_DRS_DEV_WES_VOL_DIR=${BENTOV2_WES_VOL_TMP_DIR}
 
-BENTO_DRS_TMP_VOL_DIR=${BENTOV2_ROOT_DATA_DIR}/drs/tmp
+BENTO_DRS_TMP_VOL_DIR=${BENTO_SLOW_DATA_DIR}/drs/tmp
```

Comment:
should be fast - this is for uploading files, so it's useful to be able to write to fast storage